### PR TITLE
Part 2: Editorial Fixes ODF 1.3 CS02

### PR DESCRIPTION
**Pull request by svanteschubert**
For details see: https://github.com/oasis-tcs/odf-tc/issues/21
This PR combines checkbox 4 (C4) to checkbox 9 (C9) of #21 
C4-C6 are combined as one commit as the solution was easy but the research took two days.

### CODE REVIEW

File: src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/styles.xml [hunk 1]
```diff
@@ -91,8 +91,8 @@
    <style:text-properties fo:hyphenate="false"/>
   </style:style>
   <style:style style:name="Normal" style:family="paragraph">
-   <style:paragraph-properties fo:margin-top="0.141cm" fo:margin-bottom="0.141cm" fo:hyphenation-ladder-count="no-limit"/>
-   <style:text-properties style:font-name="Liberation Sans" fo:font-family="&apos;Liberation Sans&apos;" style:font-style-name="Regular" style:font-family-generic="swiss" style:font-pitch="variable" style:font-size-complex="12pt" fo:hyphenate="false"/>
+   <style:paragraph-properties fo:line-height="normal" fo:margin-top="0.141cm" fo:margin-bottom="0.141cm" fo:hyphenation-ladder-count="no-limit"/>
+    <style:text-properties style:use-window-font-color="false" style:font-name="Liberation Sans" fo:font-family="&apos;Liberation Sans&apos;" style:font-style-name="Regular" style:font-family-generic="swiss" style:font-pitch="variable" style:font-size-complex="12pt" fo:hyphenate="false" />
```
Comment (mistmist) [inline src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/styles.xml:95]: style:use-window-font-color="false" is ignored by LO import filter, so not sure what the point is; if you want to specify a particular color use fo:color attribute

File: src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/styles.xml [hunk 2]
```diff
@@ -22,7 +22,7 @@
  </office:font-face-decls>
  <office:styles>
   <style:default-style style:family="graphic">
-   <style:graphic-properties draw:stroke="solid" svg:stroke-width="0.071cm" svg:stroke-color="#385d8a" draw:fill-color="#4f81bd" fo:wrap-option="no-wrap" draw:shadow-offset-x="0.3cm" draw:shadow-offset-y="0.3cm" draw:start-line-spacing-horizontal="0.283cm" draw:start-line-spacing-vertical="0.283cm" draw:end-line-spacing-horizontal="0.283cm" draw:end-line-spacing-vertical="0.283cm" style:flow-with-text="true"/>
```
Comment (mistmist) [inline src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/styles.xml:25]: but this should be valid?

File: src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/content.xml [hunk 3]
```diff
@@ -755,13 +755,13 @@
    <text:p text:style-name="Text_20_body">The OpenDocument specification defines conformance for packages, package consumers, and package producers, with two conformance classes called conforming and extended conforming. This chapter defines the basic requirements for the individual conformance targets.</text:p>
    <text:h text:style-name="Heading_20_2" text:outline-level="2"><text:bookmark-start text:name="__RefHeading__752789_826425813"/>Package Conformance<text:bookmark-end text:name="__RefHeading__752789_826425813"/></text:h>
    <text:h text:style-name="Heading_20_3" text:outline-level="3"><text:bookmark-start text:name="__RefHeading__752791_826425813"/><text:reference-mark-start text:name="OpenDocument Package"/>OpenDocument Package<text:bookmark-end text:name="__RefHeading__752791_826425813"/><text:reference-mark-end text:name="OpenDocument Package"/></text:h>
-   <text:p text:style-name="Text_20_body">An <text:span text:style-name="Def">OpenDocument Package</text:span> <text:span text:style-name="ISO_20_Keyword">shall</text:span> meet the following requirements:</text:p>
```
Comment (mistmist) [inline src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/content.xml]: but here it's actually removed?

File: src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/content.xml [hunk 4]
```diff
@@ -755,13 +755,13 @@
...
    <text:list xml:id="list294792977" text:style-name="Document_20_Conformance">
     <text:list-item>
-     <text:p text:style-name="P25">It <text:span text:style-name="ISO_20_Keyword">shall </text:span>be a Zip file, as defined by <text:reference-ref text:reference-format="text" text:ref-name="nref-zip">[ZIP]</text:reference-ref>. All files contained in the Zip file shall be non compressed (STORED) or compressed using the “deflate” (DEFLATED) algorithm.</text:p>
+     <text:p text:style-name="P25">It <text:span text:style-name="ISO_20_Keyword">shall</text:span> be a Zip file, as defined by <text:reference-ref text:reference-format="text" text:ref-name="nref-zip">[ZIP]</text:reference-ref>. All files contained in the Zip file <text:span text:style-name="ISO_20_Keyword">shall</text:span> be non compressed (STORED) or compressed using the “deflate” (DEFLATED) algorithm.</text:p>
     </text:list-item>
     <text:list-item>
-     <text:p text:style-name="P25">It <text:span text:style-name="ISO_20_Keyword">shall</text:span> contain a file “META-INF/manifest.xml”. This file <text:span text:style-name="ISO_20_Keyword">shall</text:span> meet the following requirements:</text:p>
```
Comment (mistmist) [inline src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/content.xml:764]: and here

File: src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/styles.xml (hunk 1 above)
Comment (svanteschubert) [reply src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/styles.xml:95]: Good to know, my intention was to overwrite the implicit parent default style. We need to reorder/redesign template styles in ODF 1.4. Would keep it for now.

File: src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/styles.xml (hunk 2 above)
Comment (svanteschubert) [reply src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/styles.xml:25]: Keep in mind, when you are reviewing commit by commit some changes might be added in the end.
For instance, the exchange of the invalid ODF XML attributes occurred very late in the commit chain.
If you want to see the complete difference from the start to the end of the patch, you need to use some other GitHub URL (I do not have at hand atm)...

File: src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/content.xml (hunk 3 above)
Comment (svanteschubert) [reply src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/content.xml]: Yes, fixed in the commit today, I overread that **every* keyword needs to be marked and assumed that no specific "*shall*" would be omitted. Fixed.

File: src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/content.xml (hunk 4 above)
Comment (svanteschubert) [reply src/main/resources/odf1.3/OpenDocument-v1.3-part2-packages_odt/content.xml:764]: Yes, there where three occurrences all the same wording. Now all fixed.

**Comment by svanteschubert** [comment]
If I am searching in the HTML syntax of part 2, I am encountering still 56 potential ISO Keywords:

- 21 x " should"
- 33 x " may"
- 2 x " need not"

Only the occurrences in non-normative/informative parts would not be highlighted.

**Pull request merged**